src/components/common/forms/ContactForm.tsx

```diff
@@ -1,28 +1,46 @@
 import React from "react";
 import { useForm } from "react-hook-form";
+import { useAppDispatch } from "@/state/app/hooks";
+import { handleModalStatus, ModalStatus } from "@/state/features";
 
 import { Button, Input, Textarea } from "@components/common";
 
 import styles from "./Forms.module.css";
 
 const ContactForm = () => {
-  const { register, handleSubmit } = useForm();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const dispatch = useAppDispatch();
+  const isFormValid = Object.keys(errors).length === 0;
```

Comment:
Is this needed? I think you could use `isValid`, from `formState`

Reply:
I like it! I also added `isSubmitted` , as `isValid` starts in `false`, and I need to show the initial inputs without error if they hasn't been submitted

```
   <span className={isSubmitted ? (isValid ? "" : styles.error) : ""}>
        Both fields are mandatory
      </span>
```